benchmark/src/androidTest/java/com/bumptech/glide/benchmark/BenchmarkFromCache.java

```diff
@@ -111,17 +112,17 @@ private void loadImageWithExpectedDataSource(
                   public boolean onLoadFailed(
                       @Nullable GlideException e,
                       Object model,
-                      Target<Bitmap> target,
+                      @NonNull Target<Bitmap> target,
                       boolean isFirstResource) {
                     return false;
                   }
 
                   @Override
                   public boolean onResourceReady(
-                      Bitmap resource,
-                      Object model,
+                      @NonNull Bitmap resource,
+                      @NonNull Object model,
```

Comment:
@sjudd Are you aware that this change could break compilation of Kotlin code?

Example fail:
https://github.com/TWiStErRob/net.twisterrob.colorfilters/actions/runs/5942409261/job/16115335917?pr=221#step:5:286

And corresponding code:
https://github.com/TWiStErRob/net.twisterrob.colorfilters/blob/0c85d9c/feature/image/src/main/kotlin/net/twisterrob/colorfilters/android/image/BitmapKeeper.kt#L111